src/test/java/org/openstreetmap/atlas/checks/validation/intersections/OceanBleedingCheckTest.java

```diff
@@ -0,0 +1,61 @@
+package org.openstreetmap.atlas.checks.validation.intersections;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Unit tests for {@link OceanBleedingCheck}
+ *
+ * @author seancoulter
+ */
+public class OceanBleedingCheckTest
+{
+    @Rule
+    public OceanBleedingCheckTestRule setup = new OceanBleedingCheckTestRule();
+
+    @Rule
+    public ConsumerBasedExpectedCheckVerifier verifier = new ConsumerBasedExpectedCheckVerifier();
+
+    @Test
+    public void invalidBuildingBleedingIntoOcean()
```

Comment:
Could you also add a check with non empty config?

Reply:
Done, see ed33586